app/src/testDeviceRelease/kotlin/de/digitalService/useID/stateMachines/SetupStateMachineTest.kt

```diff
@@ -355,7 +362,7 @@ class SetupStateMachineTest {
         }
 
         @ParameterizedTest
-        @SealedClassesSource(names = ["StartSetup", "PinReset"] , mode = SealedClassesSource.Mode.EXCLUDE, factoryClass = SetupStateFactory::class)
+        @SealedClassesSource(names = ["StartSetup", "PinReset", "AlreadySetUpConfirmation"] , mode = SealedClassesSource.Mode.EXCLUDE, factoryClass = SetupStateFactory::class)
```

Comment:
Could you please just add another test method for those invalid transitions regarding the new event `Event.ConfirmAlreadySetUp` in the inner class `InvalidTransitions`?